roles/etcd/tasks/main.yml

```diff
@@ -120,7 +120,9 @@
     enabled: yes
   register: start_result
 
-- include: etcdctl.yml
+- include_role:
+    name: etcd_common
+    tasks_from: etcdctl.yml
```

Comment:
For me, it is ok to keep the `etcdctl.yml` inside the `etcd_common` as it is a common task to do. However, I don't think it is a common tasks to do everytime the `etdc_common` role is run.

I would keep it the way it is. Once we have more "reasonable" code related to etcd client, we can stack it into the `etcd_client` role that installs the etcdctl command and drops the `etcdctl.sh` into the profiles.

I would like to introduce new etcd-client rpm later that installs just the etcdctl command. Once done, it makes sense to have `etcd_client` role.